index.js

```diff
@@ -239,32 +271,43 @@ class PearIPC extends ReadyResource {
   }
 
   unref () {
-    if (this.rawStream?.unref) this.rawStream.unref()
+    this._heartbeat?.unref()
+    this._timeout?.unref()
+    if (this.rawStream?.unref) this.rawStream?.unref()
```

Comment:
Detail: doesn't need the `?` in call to `this.rawStream.unref()`

Reply:
Likewise below on line 283

Reply:
haha thanks again